### DISCUSSION TITLE
Office_S_Tips/docx/2019/09/21/DOCX_wyszukiwanie_symboli_wieloznacznych

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#23] [Komentarze]
# Office_S_Tips/docx/2019/09/21/DOCX_wyszukiwanie_symboli_wieloznacznych

Zaawansowane wyszukiwanie i zamiana wraz z możliwością zmiany formatowania w wyszukanych fragmentach oraz przyklejania osamotnionych liter a, w, i, z, o, u na początek wiersza.

https://andrzejq.github.io/Office_S_Tips/docx/2019/09/21/DOCX_wyszukiwanie_symboli_wieloznacznych.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
